src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyoutSubItem.cs

```diff
@@ -1,47 +1,882 @@
+				spPopup.IsSubMenu = true;
+				spPopup.IsLightDismissEnabled = false;
+
+				spParentMenuFlyoutPresenter = GetParentMenuFlyoutPresenter();
+#if false // UNO TODO Windowed Popup is not available
+				if (spParentMenuFlyoutPresenter != null)
+				{
+					spParentMenuFlyout = spParentMenuFlyoutPresenter.GetParentMenuFlyout();
+					// Set the windowed Popup if the MenuFlyout is set the windowed Popup
+					if (spParentMenuFlyout && spParentMenuFlyout.IsWindowedPopup())
+					{
+						ASSERT((CPopup*)(spPopup as Popup.GetHandle()).DoesPlatformSupportWindowedPopup(DXamlCore.GetCurrent().GetHandle()));
+
+						((CPopup*)(spPopup as Popup.GetHandle()).SetIsWindowed());
+
+						// Ensure the sub menu is the windowed Popup
+						ASSERT((CPopup*)(spPopup as Popup.GetHandle()).IsWindowed());
+
+						xaml.IXamlRoot xamlRoot = XamlRoot.GetForElementStatic(spParentMenuFlyoutPresenter);
+						if (xamlRoot)
+						{
+							(spPopup as Popup.XamlRoot = xamlRoot);
+						}
+					}
+				}
+#endif
+
+				spPresenter = CreateSubPresenter();
+				spPresenterAsUI = spPresenter;
+
+				if (spParentMenuFlyoutPresenter != null)
+				{
+					int parentDepth = spParentMenuFlyoutPresenter.GetDepth();
+					(spPresenter as MenuFlyoutPresenter).SetDepth(parentDepth + 1);
+				}
+
+				spPopup.Child = spPresenterAsUI as FrameworkElement;
+
+				m_tpPresenter = spPresenter;
+				m_tpPopup = spPopup;
+
+				((ItemsControl)m_tpPresenter).ItemsSource = m_tpItems;
+
+				spPresenterAsFE = spPresenter;
+
+				spPresenterAsFE.SizeChanged += OnPresenterSizeChanged;
+				m_epPresenterSizeChangedHandler = Disposable.Create(() => spPresenterAsFE.SizeChanged -= OnPresenterSizeChanged);
+
+				m_menuHelper.SetSubMenuPresenter(spPresenter as Control);
+			}
+		}
+
+		void ForwardPresenterProperties(
+			MenuFlyout pOwnerMenuFlyout,
+			MenuFlyoutPresenter pParentMenuFlyoutPresenter,
+			MenuFlyoutPresenter pSubMenuFlyoutPresenter)
+		{
+			Style spStyle;
+			ElementTheme parentPresenterTheme;
+			object spDataContext;
+			FrameworkElement spPopupAsFE;
+			MenuFlyoutPresenter spSubMenuFlyoutPresenter = pSubMenuFlyoutPresenter;
+			Control spSubMenuFlyoutPresenterAsControl;
+			DependencyObject spThisAsDO = this;
+
+			global::System.Diagnostics.Debug.Assert(pOwnerMenuFlyout != null && pParentMenuFlyoutPresenter != null && pSubMenuFlyoutPresenter != null);
+
+			spSubMenuFlyoutPresenterAsControl = spSubMenuFlyoutPresenter;
+
+			// Set the sub presenter style from the MenuFlyout's presenter style
+			spStyle = pOwnerMenuFlyout.MenuFlyoutPresenterStyle;
+
+			if (spStyle != null)
+			{
+				((Control)pSubMenuFlyoutPresenter).Style = spStyle;
+			}
+			else
+			{
+				((Control)pSubMenuFlyoutPresenter).ClearValue(FrameworkElement.StyleProperty);
+			}
+
+			// Set the sub presenter's RequestTheme from the parent presenter's RequestTheme
+			parentPresenterTheme = pParentMenuFlyoutPresenter.RequestedTheme;
+			pSubMenuFlyoutPresenter.RequestedTheme = parentPresenterTheme;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['pSubMenuFlyoutPresenter' is null on at least one execution path.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)